### PR TITLE
BETA: Register fud.is-a.dev

### PR DESCRIPTION
**Pull request by DevMHOne**
Added `fud.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by DevMHOne** [comment]
For biolink

**Comment by PinkLittleKitty** [comment]
Vercel is not supported.

**Review by PinkLittleKitty — CHANGES_REQUESTED** [review]
Vercel is not supported.

**Comment by DevMHOne** [comment]
Check again, I've updated it

**Review by PinkLittleKitty — APPROVED** [review]

File: domains/fud.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "DevMHOne",
+        "email": "ceo@mafut.win"
+    },
+    "record": {
+        "A": "178.128.96.69"
```
Comment (CuteDog5695) [inline domains/fud.json:7]: IP should be in an array

File: domains/voucher .json
Comment (CuteDog5695) [inline domains/voucher .json:1]: there shouldnt be a space in the .json file

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by TerraPlayz** [comment]
Inactive

**Pull request closed without merging**